src/platform/BUILD.gn

```diff
@@ -85,6 +85,9 @@ if (chip_device_platform != "none" && chip_device_platform != "external") {
     # devices with multiple radios that have different sleep behavior for
     # different radios.
     chip_device_config_enable_dynamic_mrp_config = false
+
+    # Define the default endpoint id for the generic Thread network commissioning instance
+    chip_device_config_thread_network_endpoint_id = 0
```

Comment:
There should be similar requirement for Wi-Fi, shall we have a common solution for multiple networks use case.

Reply:
This definition is for the generic Thread driver instance in Platform, while most Wi-Fi driver instances are in examples. Maybe a further followup should do for this.